src/node/audio_buffer_source.rs

```diff
@@ -1,4 +1,4 @@
-use once_cell::sync::OnceCell;
+use std::cell::OnceCell;
```

Comment:
Switching from `sync` to *non-sync* here might make a difference by avoiding a mutex.